keanu-python/docs/conf.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, u'../')
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'keanu'
+copyright = u'2018, Improbable'
+author = u'Improbable'
+
+# The short X.Y version
+version = u'0.0.15'
+# The full version, including alpha/beta/rc tags
+release = u'0.0.15'
```

Comment:
Ideally yes although I'm not sure how to do this - I think it could potentially fall under SOL-2848 which is to introduce better overall versioning for python documentation.

Reply:
I don't think SOL-2848 is the right home for this, but I'll make a new ticket to cover versioning across the board (python config, python docs, java docs)